### PR TITLE
added registry-editor and registry-viewer roles info

### PR DESCRIPTION
**Pull request by gaurav-nelson**
For https://bugzilla.redhat.com/show_bug.cgi?id=1419384

### CODE REVIEW

File: install_config/registry/accessing_registry.adoc [hunk 1]
```diff
@@ -149,8 +149,9 @@ using the following command:
 - The user must have the *system:registry* role. To add this role:
```
Comment (nekop) [inline install_config/registry/accessing_registry.adoc]: This line needs to be updated too.

Comment (nekop) [reply install_config/registry/accessing_registry.adoc]: Also following `admin` and `system:image-builder` roles are not needed.

Comment (gaurav-nelson) [reply install_config/registry/accessing_registry.adoc]: @nekop 
1. For `The user must have the *system:registry* role. To add this role:` this line needs update too? what changes are required? 

2. For `Also following admin and system:image-builder roles are not needed.` Do you mean to remove the following lines?
```
- For writing or pushing images, for example when using the `docker push` command,
the user must have the *system:image-builder* role. To add this role:
+
----
$ oc adm policy add-role-to-user system:image-builder <user_name>
----
```

Comment (nekop) [reply install_config/registry/accessing_registry.adoc]: The "system:" prefix roles are meant for system service account. It's not for normal users.

For image pull, we need "registry-viewer". For push "registry-editor". Other roles are not needed.

**Comment by gaurav-nelson** [comment]
@nekop Thanks for the comments, please see the updates.

**Comment by nekop** [comment]
Still needs changes:

- `oc adm policy` (`oc adm` is cluster-admin command) is not required as it's a project level roles, so `oc policy` is enough
- `admin` role is not required for docker pull/push

What we need is:

For pull:

```
oc policy add-role-to-user registry-viewer <user_name>
```

For push:

```
oc policy add-role-to-user registry-editor <user_name>
```

That's all.

File: install_config/registry/accessing_registry.adoc [hunk 2]
```diff
@@ -146,11 +146,14 @@ using the following command:
...
++
+To add this role:
 +
 ----
-# oc adm policy add-role-to-user system:registry <user_name>
+# oc adm policy add-role-to-user registry-editor <user_name> -n <project> <1>
 ----
+<1> You can also add the role `registry-viewer` for providing pull only access to the registry.
 
 - Have the *admin* role for the project associated with the Docker operation. For
```
[the remark this answers is not in the document]
Comment (gaurav-nelson) [reply install_config/registry/accessing_registry.adoc]: @nekop I am updating the other two points as follows:
```
- For pulling images, for example when using the `docker pull` command,
the user must have the *registry-viewer* role. To add this role:
+
----
$ oc adm policy add-role-to-user registry-viewer <user_name>
----

- For writing or pushing images, for example when using the `docker push` command,
the user must have the *registry-editor* role. To add this role:
+
----
$ oc policy add-role-to-user registry-editor <user_name>
----
```

[the remark this answers is not in the document]
Comment (nekop) [reply install_config/registry/accessing_registry.adoc]: Please remove the `admin` part, it's a misleading example. We can use registry-editor for that purpose.

**Comment by gaurav-nelson** [comment]
Thank you @nekop I have removed the admin parts int the latest commit.

**Comment by gaurav-nelson** [comment]
@nekop PTAL, Thank you for looking at this yesterday. I can merge this if there are no more changes.

File: install_config/registry/accessing_registry.adoc [hunk 3]
```diff
@@ -146,24 +146,18 @@ using the following command:
...
 ----
-# oc adm policy add-role-to-user system:registry <user_name>
-----
-
-- Have the *admin* role for the project associated with the Docker operation. For
-example, if accessing images in the global *openshift* project:
-+
-----
- $ oc adm policy add-role-to-user admin <user_name> -n openshift
+$ oc adm policy add-role-to-user registry-viewer <user_name>
```
Comment (nekop) [inline install_config/registry/accessing_registry.adoc]: This should be "oc policy", drop the "adm".

**Comment by nekop** [comment]
One minor change needed, the rest LGTM.

**Pull request merged**